File: lib/models/post.dart
class PostModel {
  final String title, content, author, createdAt;
  final String? filePath;
  final int id, userId;
  void showInfo() {
    print("$id, $title, $content, $author, $filePath,  $userId, $createdAt");
  }

  PostModel.fromDB(Map<String, dynamic> db)
      : id = db['id'],
        title = db['title'],
        content = db['content'],
        author = db['author'],
        filePath = db['file_path'],
        userId = db['userId'],
        createdAt = db['createdAt'].toString();
}

File: models/post.js
const Sequelize = require("sequelize");

const sequelize = require("../util/database");

const Post = sequelize.define("post", {
  id: {
    type: Sequelize.INTEGER,
    autoIncrement: true,
    allowNull: false,
    primaryKey: true,
  },
  title: { type: Sequelize.STRING, allowNull: false },
  content: { type: Sequelize.TEXT, allowNull: false },
  author: { type: Sequelize.STRING, allowNull: false },
  userId: { type: Sequelize.INTEGER, allowNull: false },
  file_path: { type: Sequelize.STRING, allowNull: true },
});

module.exports = Post;
